src/helpers/platform.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Check if current platform is a mobile device
+ *
+ * @return {boolean} whether the platform is a mobile device
+ */
+export function isMobile() {
+	// Use client hints if already available
+	if (navigator?.userAgentData?.mobile !== undefined) return navigator.userAgentData.mobile
+
+	// use regex to match userAgent (required for Safari and Firefox in 2022)
+	const mobileDevices = [
+		/Android/i,
+		/webOS/i,
+		/iPhone/i,
+		/iPad/i,
+		/iPod/i,
+		/playbook/i,
+		/silk/i,
+		/BlackBerry/i,
+		/Windows Phone/i,
+	]
+
+	return mobileDevices.some(regex => navigator.userAgent.match(regex))
+}
```

Comment:
I think adding an alternative definition of `isMobile` is confusing.

My understanding is that `isMobile` is being used in a number of different nextcloud apps and usually it means has a small screen. Maybe we should rename the original. But I'd be tempted to leave the definition as is for now.

Is this adjustment of `isMobile` still needed now that mobile and desktop behave almost the same?

Reply:
There are no keyboard shortcuts on real mobile devices (IPhone, Android...) as there are no meta keys, but "small screen size" could also be a desktop device with small sized browser window.

Reply:
The `Editor` component provides is mobile and then it's also getting [used](https://github.com/nextcloud/text/blob/879eaec610f310b3bc4e695f662eb88fe56563eb/src/components/Editor/Status.vue#L25) by the `Editor/Status` component.

In that context it hides the status and i think that is so it fits on a smaller screen.

All other uses that i found hide keyboard shortcuts - which as you explained makes sense on mobile devices.

How about calling the function here something like `hasKeyboardShortcuts` or `displayShortcuts` or `hasKeyboard`?
I think the `isMobile` name is choosen badly and `hasSmallScreen` would be better - but I would not touch taht for now.

Reply:
Thanks for following up on this. I think this is resolved now.